src/mongo_logger.ts

```diff
@@ -281,7 +282,7 @@ export interface Log extends Record<string, any> {
 
 /** @internal */
 export interface MongoDBLogWritable {
-  write(log: Log): void;
+  write(log: Log): any;
```

Comment:
```suggestion
  write(log: Log): PromiseLike<unknown> | unknown;
```

If possible I think we would prefer `unknown` here, it forces you to write type assertion/narrowing code to make sure we're handling the right value. I also think there's an informational benefit to indicating the result can be a promise, thoughts?